crates/rooch-rpc-api/src/api/eth_api.rs

```diff
@@ -87,21 +101,25 @@ pub trait EthAPI {
 
     /// Sends signed transaction, returning its hash.
     #[method(name = "eth_sendRawTransaction")]
-    async fn send_raw_transaction(&self, bytes: Bytes) -> RpcResult<H256>;
+    async fn send_raw_transaction(&self, bytes: Bytes) -> RpcResult<H256View>;
 
     /// Returns transaction receipt by transaction hash.
     #[method(name = "eth_getTransactionReceipt")]
-    async fn transaction_receipt(&self, hash: H256) -> RpcResult<Option<TransactionReceipt>>;
+    async fn transaction_receipt(&self, hash: H256View) -> RpcResult<Option<TransactionReceipt>>;
 
     /// Get transaction by its hash.
-    #[method(name = "eth_getTransactionByHash")]
-    async fn transaction_by_hash(&self, hash: H256) -> RpcResult<Option<Transaction>>;
+    #[method(name = "eth_getTransactionByHashAndIndex")]
```

Comment:
There is no RPC API `eth_getTransactionByHashAndIndex` in Ethereum RPC APIs.

Reply:
https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_gettransactionbyblocknumberandindex

Reply:
There's a `eth_getTransactionByBlockHashAndIndex`: https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_gettransactionbyblockhashandindex